cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -247,6 +392,34 @@ func isRepoReady(repoUnstructured *unstructured.Unstructured) (bool, error) {
 	return false, nil
 }
 
+//
+// TODO the semantics of this really should be do we need to keep polling or not
+//
+func isChartPullComplete(unstructuredChart *unstructured.Unstructured) (bool, error) {
```

Comment:
As above, hopefully this fn isn't required if we can only list charts that we know are ready.